ios/Codeforces Watcher/Application/features/users/UserViewController.swift

```diff
@@ -48,16 +48,35 @@ class UserViewController: UIViewControllerWithCross {
         navigationItem.rightBarButtonItem = UIBarButtonItem(
             image: UIImage(named: "removeIcon"), 
             style: .plain, 
-            target: self, 
-            action: #selector(removeTapped)
+            target: self,
+            action: #selector(showAlertToDeleteUser)
         )
         
         buildViewTree()
         setConstraints()
         bind()
     }
     
-    @objc func removeTapped() {
+    @objc func showAlertToDeleteUser() {
+        let alertController = UIAlertController(
+            title: "Delete user",
+            message: "Are you sure you want to delete \(user.handle)?",
+            preferredStyle: .alert
+        )
+        
+        let okButton = UIAlertAction(title: "OK", style: .cancel) {
+            UIAlertAction in
```

Comment:
4 minor things to fix and remember here:
1. Variable names starts with a small letter -> `uiAlertAction`.
2. Arguments to lambdas (unnamed functions / methods) are usually written right after `{` -> `{ uiAlertAction in`.
3. If arguments aren't used, they can be ignored by using instead of name `_` -> `{ _ in`.
4. If there is just one argument, in some cases you can omit the whole thing:

```
let okButton = UIAlertAction(title: "OK", style: .cancel) {
    self.removeTapped()
}
```